lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.tsx

```diff
@@ -99,12 +92,6 @@ class KeyboardAccessoryView extends Component<KeyboardAccessoryViewProps> {
 
     this.registerForKeyboardResignedEvent();
     this.registerAndroidBackHandler();
-
-    if (props.iOSScrollBehavior) {
-      LogService.warn(`The 'Keyboard.KeyboardAccessoryView' component's prop 'iOSScrollBehavior' is deprecated. 
```

Comment:
`LogService` can be removed from the imports

Reply:
Yes, that was the issue that failed my build... Thanks